### ISSUE TITLE
one adjustment to the design - photos in the 'Additional Info' section

### ISSUE THREAD

**Issue by ryanbranciforte** [trailheadlabs/carlessca#8]
I'd like to have some photos in the "Additional Info" section (screen shot below). These photos will be more informational, things like the getting on the amtrak bus, the yarts/amtrak transit center, putting your bike on the bus, transit stop in Yosemite and Tahoe. The photos will help folks visualize the transit component of the trip. Make sense? 

I wasn't able to get these in the design as I thought of it after the last round of changes. 

![screen shot 2015-12-14 at 2 32 25 pm](https://cloud.githubusercontent.com/assets/2602563/11904645/430fdc64-a576-11e5-9d77-d1d65a3b4bb1.png)

**Comment by jmoe** [comment]
We can try to put some images under the 1,2,3 but let's try to freeze the design now so we're aren't try to hit a moving target with implementation for Beta.

**Comment by ryanbranciforte** [comment]
agreed. this was the only update to the original design that i came across. yeah, i think under 1,2,3  or under cost section makes sense.

**Comment by jmoe** [comment]
This feels like an enhancement at this point, I suggest we bikerack it.
